draft-ietf-httpbis-digest-headers.md

```diff
@@ -375,10 +367,14 @@ Want-Content-Digest: sha-512;q=0.3, sha-256;q=1, unixsum;q=0
 
 Digest-algorithm values are used to indicate a specific digest computation.
 
-~~~
+~~~ abnf
    digest-algorithm = token
 ~~~
 
+When it is convenient to do so,
```

Comment:
I'd like to see some indication of what would make it "convenient" here -- this is as unclear as "incremental" was before.

Reply:
Probably it's better to mention incremental algorithms then :P

```suggestion
When using algorithms based on incremental computation,
(see {{?RFC1624}}) 
```

Reply:
alternatively just pass the ball to implementers.

```suggestion
When implementers deem convenient to do so,
```

Reply:
@LPardue if none of the above solutions work, I'd just remove these 3 lines since checksum implementers already know that :)

Reply:
lets just delete it